DOCUMENTATION.md

```diff
@@ -405,6 +405,78 @@ You can omit this argument if you do not require a reference for the notificatio
 
 </details>
 
+### Precompiled letter
+
+This is an invitation-only feature. Contact the GOV.UK Notify team on the [support page](https://www.notifications.service.gov.uk/support) or through the [Slack channel](https://ukgovernmentdigital.slack.com/messages/govuk-notify) for more information.
+
+#### Method
+
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
+
+```csharp
+LetterNotificationsResponse response = client.SendPrecompiledLetter(
+    clientReference,
+    pdfContents
+    );
+```
+
+</details>
+
+#### Arguments
+
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
+
+##### clientReference (required)
+
+A unique identifier you create. This reference identifies a single unique notification or a batch of notifications. It must not contain any personal information such as name or postal address.
+
+##### pdfContents (required for the SendPrecompiledLetter method)
+
+The precompiled letter must be a PDF file. The method sends the contents of the file to GOV.UK Notify.
+
+```csharp
+byte[] pdfContents = File.ReadAllBytes("<PDF file path>");
+```
+
+</details>
+
+#### Response
+
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
+
+If the request to the client is successful, the client returns a `LetterNotificationResponse` with only `id` and `reference` set:
```

Comment:
Set with what? What does set mean in this context? Are there other things that could be set? If so, what are they? Why is it only these two that are set?

Reply:
same object returned as other responses, only most fields are null as template has not been used because it is a precompiled letter